solution1.py

```diff
@@ -0,0 +1,34 @@
+# https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/
+# // Time Complexity : o(n)
+# // Space Complexity : o(1)
+# // Did this code successfully run on Leetcode :yes
+# // Any problem you faced while coding this : maintaining three pointers
```

Comment:
Fast is incremented no matter what, instead of maintaining three pointers you could use a for loop. Bringing down to manual track of two pointers!